dict.js

```diff
@@ -44,14 +44,17 @@ Dict.prototype.assertString = function (key) {
     if (typeof key !== "string") {
         throw new TypeError("key must be a string but Got " + key);
     }
-}
+};
 
 Dict.prototype.get = function (key, defaultValue) {
     this.assertString(key);
-    var mangled = mangle(key);
-    if (mangled in this.store) {
-        return this.store[mangled];
-    } else if (arguments.length > 1) {
+    if (key == '__proto__') {
+        key = '$__proto__';
```

Comment:
Note that while this adds "__proto__" to the domain of valid keys, it now overlaps with "$__proto__". Consider either always adding the "_" or "$" prefix (as before) (they are both valid characters in JavaScript identifiers), or consistently adding an affix to all names with a common substring. If you decide not to use Object.create(null), as I recommend, consider the effect of enumerable properties added to Object.prototype and decide whether that concerns any users of the Collections library.